config.js

```diff
@@ -55,6 +55,9 @@ module.exports = {
     emailTemplateId: get('NOTIFY_EMAIL_TEMPLATE', 'not_email', requiredInProduction),
     healthCheckUrl: process.env.NOTIFY_HEALTH_CHECK_URL || 'https://api.notifications.service.gov.uk/_status',
   },
+  notifcationService: {
+    url: get('NOTIFICATION_SERVICE_URL', 'http://localhost:9191/notifications-service'),
```

Comment:
Its a preference, but this URL will be the same for everything we take out of the FE database. 
I guess i'm asking are you planning on having NOTIFICATION_SERVICE, SHIFTS_SERVICE, PREFERENCE_SERVICE etc otherwise go with CMD_API_URL or something.

Reply:
Ahh that makes sense, I'll go with the latter.